src/components/Button/Button.tsx

```diff
@@ -52,7 +57,11 @@ export const Button: FC<ButtonProps> = ({
       ...buttonProps
     },
     <>
-      <div className="zui-button-content">{isLoading ? <Spinner className="zui-button-spinner" /> : children}</div>
+      <div className="zui-button-content">
+        {startEnhancer}
+        {isLoading ? <Spinner className="zui-button-spinner" /> : children}
```

Comment:
Should the loading state still show the icon? I haven't seen the design but that feels weird to me.